cli/cmd/summary.go

```diff
@@ -62,3 +73,86 @@ func checkFilePath(filename string) error {
 	dirPath := filepath.Dir(filename)
 	return os.MkdirAll(dirPath, 0755)
 }
+
+func generateSummaryTable() *schema.Table {
+	tableName := "cloudquery_sync_summary"
```

Comment:
`cloudquery_sync_summary` -> `cloudquery_sync_summaries`, for consistency with our resource naming convention? 😄